### DISCUSSION TITLE
@livewireStyles @livewire('navigation-menu') showing on top of page after install

### DISCUSSION THREAD

**Discussion by acharseth** [livewire/livewire#4903] [1. Help] [answered]
I have just installed Laravel with Livewire.
When I run php artisan serve and then enter localhost:8000 and login then I get:

_@livewireStyles 
@livewire('navigation-menu')_ 

on top of the Dashbord page (see screenshot below).

And there is no menu top right corner.

I installed Laravel with Livewire like this:

	~/.composer/vendor/bin/laravel new jetstream-eksempel
	cd jetstream-eksempel
	php ../composer.phar require laravel/jetstream
	php artisan jetstream:install livewire --teams
	npm install
	npm run dev
	php artisan migrate

Just in case `php artisan jetstream:install livewire --teams` only install the team-part (not Livewire itself(?)) I have also run `php artisan jetstream:install livewire` (afterwards).

I have tried following all the tips on https://forum.laravel-livewire.com/t/resolved-livewirescripts-and-livewirestyles-showing-on-page/147 but get the same result. Summed up here:

1. Minor change in files containing these.
(Added space, saved, removed space, save. Assuming compilation happens automatically after this)
2. `composer dumpautoload` and `php artisan config:cache`
3. Added `()` behind `@livewireStyles`

I wonder if livewire is correctly installed or installed at all.
The response after installing livewire looks like this:

	% php artisan jetstream:install livewire
	Migration created successfully.
	sh: line 0: exec: composer: not found
	Copied Directory [/vendor/laravel/sanctum/database/migrations] To [/database/migrations]
	Copied File [/vendor/laravel/sanctum/config/sanctum.php] To [/config/sanctum.php]
	Publishing complete.
	
	Livewire scaffolding installed successfully.
	Please execute "npm install && npm run dev" to build your assets.

I notice the message `sh: line 0: exec: composer: not found`.
I do not have a path to composer but run `php ../composer.phar` instead. Could this be the issue?

I am new to Laravel so all help much appreciated.

Screenshot:
<img width="777" alt="image" src="https://user-images.githubusercontent.com/16928675/164102422-e44390b6-8b73-425f-b661-a9661b5364d8.png">

**Comment by tushargugnani** [comment]
Try clearing out the cache. `php artisan optimize:clear`

**Reply by acharseth** [reply]
Thanks for this advice, @tushargugnani , I have tried this as well (and the variant `php artisan config:cache`). Same result :-(

**Comment by acharseth** [comment, marked as answer]
Yes, composer needs to be in the path for `php artisan` command to work it seems (alias is not sufficient). So I copied composer.phar to /usr/local/bin/composer and now I get another error message:
_Attempt to read property "name" on null_ on `{{ Auth::user()->currentTeam->name }}` so I guess something is wrong with my teams setup, probably because I already had a user defines before I installed teams so this user had no teams belonging.

I fixed this by deleting all Laravel tables and reinstalled them with command `php artisan migrate`.

**Comment by aktil** [comment]
Hello, i have the same problem 
and i coudn't understand you. Can you clearly explain me

**Reply by acharseth** [reply]
The problem for me was that Livewire was not properly installed. I had chosen not to make «composer» a runable command (so I had to run « php composer.phar …»). After making «composer» a runable command and running the installation again it worked fine.
So I guess the bottom line, if you have a similar issue, is to check if Livewire was properly installed. Check the output of the installation for error or warnings messages. If you can not find the problem then I guess you should raise your own issue on GitHub.
